src/signals/incident/components/form/ContainerSelect/LegendToggleButton/LegendToggleButton.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+
+import { MapPanelContext } from '@amsterdam/arm-core';
+import { MapLayers } from '@amsterdam/asc-assets';
+import { Button } from '@amsterdam/asc-ui';
+import { SnapPoint } from '@amsterdam/arm-core/lib/components/MapPanel/constants';
+
+const ICON_SIZE = 20;
+
+export interface LegendToggleButtonProps {
+  isRenderingLegendPanel: boolean;
+  onClick: () => void;
+}
+
+const StyledButton = styled(Button)`
+  box-sizing: border-box;
```

Comment:
Why is this needed? Is not the default value?

Reply:
Discussed and resolved